manifest.json

```diff
@@ -90,7 +110,18 @@
               "tags": {
                 "4.6.2-$(RuntimeReleaseDateStamp)-windowsservercore-ltsc2016": {},
                 "4.6.2-windowsservercore-ltsc2016": {}
-              }
+              },
+              "customBuildLegGrouping": [
+                {
+                  "name": "pr-build",
+                  "dependencies": [
+                    "$(Repo:runtime):4.7.1-windowsservercore-ltsc2016",
```

Comment:
Where is the dependency on the 4.7.1 runtime coming from?  Isn't the dependency just on the sdk?  This comment applies to other groupings as well.

Reply:
The dependency graph needs to be flattened because of how Image Builder works.  Ideally, we should just add a dependency on the SDK and get all of it's dependencies along with it, but that's not possible with the current logic.  I'll log a bug on Image Builder.

Reply:
https://github.com/dotnet/docker-tools/issues/175